metrics_test.go

```diff
@@ -44,21 +43,21 @@ func TestLatencyEWMA(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	exp := 100.0
-	mu := exp
-	sig := 10.0
-	next := func() time.Duration {
-		mu := (rand.NormFloat64() * sig) + mu
```

Comment:
This was the problem. This would generate values outside of the range [90, 110] due to float overflows.